ecgtools/parsers/cesm.py

```diff
@@ -103,6 +107,7 @@ def parse_cesm_history(file):
                 for v, da in ds.variables.items()
                 if time in da.dims and v not in {time, time_bounds}
             ]
+            info['frequency'] = ds.attrs['time_period_freq']
```

Comment:
Thank you, @mgrover1! Can we confirm whether this `time_period_freq` attribute is *only* available in CESM2+? As far as I can remember from my conversation with @sherimickelson, older versions of CESM didn't output this attribute across all components.